tools/compute_cmvn_stats.py

```diff
@@ -24,14 +24,16 @@ def __call__(self, batch):
         number = 0
         for item in batch:
             key = item[0]
-            waveform, sample_rate = torchaudio.load_wav(item[1])
-            mat = kaldi.fbank(waveform,
-                              num_mel_bins=self.feat_dim,
-                              dither=0.0,
-                              energy_floor=0.0)
-            mean_stat += torch.sum(mat, axis=0)
-            var_stat += torch.sum(torch.square(mat), axis=0)
-            number += mat.shape[0]
+            speed_perturb = [1.0]
```

Comment:
Refine this if you want to add speed support in this tool,
Or remove speed perturb related change